Today/TodayViewController.swift

```diff
@@ -37,12 +37,17 @@ class TodayViewController: UIViewController, NCWidgetProviding {
 
 extension TodayViewController: LocationTrackerDelegate {
 
-    func locationTracker(_ locationTracker: LocationTracker, didFailWith error: Error) {
+    func didFailWithError(_ error: Error, on locationTracker: LocationTracker) {
         // ignore errors in the widget
     }
 
-    func locationTracker(_ locationTracker: LocationTracker, didFindLocation placemark: CLPlacemark) {
-        Location.current = Location(placemark: placemark)
+    func didFindLocation(_ placemark: CLPlacemark, on locationTracker: LocationTracker) {
+//        Location.current = Location(placemark: placemark)
```

Comment:
Why is this commented out?

Reply:
This is all the widget stuff that is also based on the same stuff the app is, so will need to update that as well in the future.